Meadowlark-js/tests/http/local.33b.http

```diff
@@ -181,12 +182,12 @@ authorization: bearer {{authToken4}}
 
 
 ### 403 deleting with wrong owner
-DELETE http://localhost:3000/local/v3.3b/ed-fi/students/WhT14ozRv-M80122ZFtMigb_5VtptJfuO4KRKA
+DELETE http://localhost:3000/local/v3.3b/ed-fi/students/{{students.response.headers.location}}
 authorization: bearer {{authToken1}}
 
 
 ### 204 deleting with correct owner
-DELETE http://localhost:3000/local/v3.3b/ed-fi/students/WhT14ozRv-M80122ZFtMigb_5VtptJfuO4KRKA
+DELETE http://localhost:3000/local/v3.3b/ed-fi/students/{{students.response.headers.location}}
 authorization: bearer {{authToken4}}
 
 ###
```

Comment:
Please also adjust test `#### Contains two different school years` to capture the location header so that it can be used in `### Solved in RND-426`